back/src/main/java/com/speakerspace/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,141 @@
+        logger.error("Null pointer exception: {}", ex.getMessage(), ex);
+
+        Map<String, Object> errorResponse = new HashMap<>();
+        errorResponse.put("error", "Data validation error");
+        errorResponse.put("message", "Required data is missing or invalid");
+
+        return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(errorResponse);
+    }
+
+    @ExceptionHandler(IllegalArgumentException.class)
+    public ResponseEntity<Map<String, Object>> handleIllegalArgumentException(IllegalArgumentException ex) {
+        logger.error("Illegal argument/validation error: {}", ex.getMessage(), ex);
+
+        Map<String, Object> errorResponse = new HashMap<>();
+        errorResponse.put("error", "Invalid input");
+        errorResponse.put("message", ex.getMessage());
+
+        return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(errorResponse);
+    }
+
+    @ExceptionHandler(UnsupportedOperationException.class)
+    public ResponseEntity<Map<String, Object>> handleUnsupportedOperation(UnsupportedOperationException ex) {
+        logger.error("Unsupported operation error: {}", ex.getMessage(), ex);
+
+        Map<String, Object> errorResponse = new HashMap<>();
+        errorResponse.put("error", "Operation not supported");
+        errorResponse.put("message", "An internal error occurred while processing your request");
+
+        return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(errorResponse);
+    }
+
+    @ExceptionHandler(AccessDeniedException.class)
+    public ResponseEntity<Map<String, Object>> handleAccessDenied(AccessDeniedException ex) {
+        logger.warn("Access denied: {}", ex.getMessage());
+
+        Map<String, Object> errorResponse = new HashMap<>();
+        errorResponse.put("error", "Access denied");
+        errorResponse.put("message", "You do not have permission to perform this action.");
+
+        return ResponseEntity.status(HttpStatus.FORBIDDEN).body(errorResponse);
+    }
+
+    @ExceptionHandler(EntityNotFoundException.class)
+    public ResponseEntity<Map<String, Object>> handleEntityNotFound(EntityNotFoundException ex) {
+        logger.warn("Entity not found: {}", ex.getMessage());
+
+        Map<String, Object> errorResponse = new HashMap<>();
+        errorResponse.put("error", "Resource not found");
+        errorResponse.put("message", ex.getMessage());
+
+        return ResponseEntity.status(HttpStatus.NOT_FOUND).body(errorResponse);
+    }
+
+    @ExceptionHandler(ValidationException.class)
+    public ResponseEntity<Map<String, Object>> handleValidationException(ValidationException ex) {
+        logger.error("Validation error: {}", ex.getMessage());
+
+        Map<String, Object> errorResponse = new HashMap<>();
+        errorResponse.put("error", "Validation failed");
+        errorResponse.put("message", ex.getMessage());
+        errorResponse.put("details", ex.getErrors());
+
+        return ResponseEntity.badRequest().body(errorResponse);
```

Comment:
suggestion: créer un objet dédié plutôt qu'une Map

```Java
public record AppError(String error, String message, Object details) {
   public AppError(String error, String message) {
       return this(error, message, null);
   }
}
```

```suggestion
        return ResponseEntity.badRequest()
            .body(new AppError("Validation failed",ex.getMessage(),ex.getErrors()));
```

Éventuellement avec un `@Builder` sur le record `AppError` comme tu préfères.